File: MiniDOMAsteriskLib/CallManagers/Action/ParkedCalls.cs
using minidom.CallManagers.Responses;

namespace minidom.CallManagers.Actions
{
    public class ParkedCalls : Action
    {
        public ParkedCalls() : base("ParkedCalls")
        {
        }

        protected override ActionResponse AllocateResponse(string[] rows)
        {
            return new ParkedCallsResponse(this);
        }
    }
}

File: MiniDOMAsteriskLib/CallManagers/Action/ParkedCalls.vb
Imports System.Net.Sockets
Imports System.Threading
Imports System.Net
Imports minidom.CallManagers.Responses

Namespace CallManagers.Actions

    Public Class ParkedCalls
        Inherits Action

        Public Sub New()
            MyBase.New("ParkedCalls")
        End Sub

        Protected Overrides Function AllocateResponse(rows() As String) As ActionResponse
            Return New ParkedCallsResponse(Me)
        End Function


    End Class

End Namespace